katago_server/runs/viewsets/run.py

```diff
@@ -31,3 +31,14 @@ def current(self, request):
             return Response({'error': 'No active run.'}, status=status.HTTP_404_NOT_FOUND)
         self.kwargs["pk"] = current_run.pk
         return self.retrieve(request)
+
```

Comment:
Maybe you can remove the current as it is no longer needed.

Reply:
ok